### DISCUSSION TITLE
A bug in list name handling? Or just API 1.1 stuff?

### DISCUSSION THREAD

**Discussion by NetwarSystem** [tweepy/tweepy#1640] [Questions] [answered]
I have a bunch of simple command line scripts for doing things with Twitter. This code fragment is from one that just prints information about the lists associated with a given account. The older lists, fire and geopol, are from late last year. The two new ones, mine and test, were created this evening.  The older lists have a proper slug, the newer ones just have their numeric IDs.

There are other scripts for checking list membership, for adding and removing accounts, and this change to how the slug works is a big hassle. Instead of the fairly descriptive slug, newer lists require the numeric ID. I suppose I could add some name to numeric ID step to my Twitter bag of tricks library, but this just looks wrong. Is this an inescapable effect of the change to API 1.1?

```
lists = api.lists_all(name)
for list in lists:
   print(list.full_name + "," + list.slug + "," + list.id_str + "," + list.name + "," + list.description)

# older lists
#@AmericanHacker/fire-14770,fire-14770,1391254253092540418,fire,
#@AmericanHacker/geopol-57545,geopol-57545,1383115540256878593,geopol,

# new test lists
#@AmericanHacker/1427950317878931468,1427950317878931468,1427950317878931468,mine,My list
#@AmericanHacker/1427889397064683524,1427889397064683524,1427889397064683524,test,

# 'add_member',
# 'created_at',
# 'description',
# 'destroy',
# 'following',
# 'full_name',
# 'id',
# 'id_str',
# 'is_member',
# 'is_subscribed',
# 'member_count',
# 'members',
# 'mode',
# 'name',
# 'parse',
# 'parse_list',
# 'remove_member',
# 'slug',
# 'subscribe',
# 'subscriber_count',
# 'subscribers',
# 'timeline',
# 'unsubscribe',
# 'update',
# 'uri',
# 'user'

```

**Comment by Harmon758** [comment, marked as answer]
This seems to be a change in Twitter's API, rather than an issue with Tweepy:
https://twittercommunity.com/t/list-slug/155129

If you have further questions about it, you should ask on the [Twitter Developers Forums](https://twittercommunity.com/).
